streams/src/main/java/org/apache/kafka/streams/kstream/ValueMapper.java

```diff
@@ -35,13 +35,13 @@
  * @see KTable#mapValues(ValueMapper)
  */
 @InterfaceStability.Unstable
-public interface ValueMapper<V, VR> {
+public interface ValueMapper<K, V, VR> {
 
     /**
      * Map the given value to a new value.
```

Comment:
The 1-liner description should be updated to match the new (key, val) -> newVal behavior.